packages/market-service/src/exchange-rates-host/exchange-rates-host.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  FiatMarketDataArgs,
+  FiatPriceHistoryArgs,
+  HistoryData,
+  HistoryTimeframe,
+  MarketData
+} from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+
+import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
+import { bnOrZero } from '../utils/bignumber'
+import { isValidDate } from '../utils/isValidDate'
+import { rateLimitedAxios } from '../utils/rateLimiters'
+import { ExchangeRateHostHistoryData, ExchangeRateHostRate } from './exchange-rates-host-types'
+
+const axios = rateLimitedAxios(RATE_LIMIT_THRESHOLDS_PER_MINUTE.DEFAULT)
+const baseCurrency = 'USD'
+const baseUrl = 'https://api.exchangerate.host'
+
+export const findByFiatSymbol = async ({
+  symbol
+}: FiatMarketDataArgs): Promise<MarketData | null> => {
+  try {
+    const { data } = await axios.get<ExchangeRateHostRate>(
+      `${baseUrl}/latest?base=${baseCurrency}&symbols=${symbol}`
+    )
+    const result = data as ExchangeRateHostRate
+    // we only need the price key in the `web`
+    return {
+      price: result.rates[symbol].toString(),
+      marketCap: '0',
+      changePercent24Hr: 0,
+      volume: '0'
+    }
+  } catch (e) {
+    console.warn(e)
+    throw new Error('FiatMarketService(findByFiatSymbol): error fetching market data')
+  }
+}
+
+export const findPriceHistoryByFiatSymbol = async ({
+  symbol,
+  timeframe
+}: FiatPriceHistoryArgs): Promise<HistoryData[]> => {
+  const end = dayjs().startOf('day')
+  let start
+  switch (timeframe) {
```

Comment:
This is a bit boilerplaty but after playing with TSPlayground trying to get the enum keys and a default, I couldn't come up with a better way to achieve this.